addons/website/static/src/scss/website.scss

```diff
@@ -416,7 +416,7 @@ font[class*='bg-'] {
 
 // Probably outdated
 // Disable fixed height
-@media (max-width: 400px) {
+@include media-breakpoint-down(sm) {
```

Comment:
I think this line will be correct with BS5 but before BS5 we were giving the value of the next breakpoint instead of the breakpoint itself. So here the CSS generates `@media (max-width: 768px)` and that's not what we want if I understood the commit message correctly. So I think it should be changed here and we should be careful to adapt it to the correct version.

Reply:
Also in the task we talk about the "image carousel" and here in the commit we talk about the "image gallery". It might be good to modify the commit message to explain that the fix is ​​more generic and does not only concern "image gallery".